android/app/src/main/java/com/swmansion/gesturehandler/react/RNZoomableButtonManager.java

```diff
@@ -64,9 +64,9 @@ public boolean onTouchEvent(MotionEvent ev){
                 if (mIsTaskScheduled) {
                     onReceivePressEvent(false);
                 }
+                mIsTaskScheduled = false;
```

Comment:
what's this changing?

Reply:
seems like it was onReceivePressEvent was triggering twice on press on android, moving that check immediately after the onReceivePressEvent call blocked a second call